models/articles-models.js

```diff
@@ -12,3 +13,18 @@ ORDER BY articles.created_at DESC;`;
     return rows;
   });
 };
+
+exports.fetchArticleById = (article_id) => {
+  const articleQueryString = `
+SELECT article_id, author, topic, title, body,
+created_at, votes, article_img_url
+FROM articles
+WHERE article_id = $1;`;
+  if (typeof article_id !== 'number' || isNaN(article_id))
```

Comment:
You should get a psql error if the article_id is `NaN` so this shouldn't really need a custom error.